README.md

```diff
@@ -117,8 +117,22 @@ To load JDWP agent for UAA jvm debugging, start the server as follows:
 ```sh
```

Comment:
How about removing the old option so we can simplify the document?

Reply:
Old option meaning using `spring.profiles.active` property? I want to keep it so one can just use spring profile with all the desired options instead.